modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/util/test/GroupUtilTest.java

```diff
@@ -94,17 +97,24 @@ public void testGetDepotGroupIdWithFF() throws Exception {
 
 	@Test
 	public void testGetGroupId() throws Exception {
-		Group group = _groupLocalService.getGroup(TestPropsValues.getGroupId());
+		_testGetGroupId(
+			_groupLocalService.getGroup(TestPropsValues.getGroupId()));
+	}
 
-		Assert.assertEquals(
-			Long.valueOf(group.getGroupId()),
-			GroupUtil.getGroupId(
-				group.getCompanyId(), group.getGroupKey(), _groupLocalService));
-		Assert.assertEquals(
-			Long.valueOf(group.getGroupId()),
-			GroupUtil.getGroupId(
-				group.getCompanyId(), String.valueOf(group.getGroupId()),
-				_groupLocalService));
+	@Test
+	public void testGetGroupIdForUserGroup() throws Exception {
+		UserGroup userGroup = null;
+
+		try {
+			userGroup = UserGroupTestUtil.addUserGroup();
```

Comment:
I think you could move this out of the block and only apply the finally block if the _testGetGroupId fails